ObjCVoiceQuickstart/ViewController.m

```diff
@@ -246,11 +247,21 @@ - (void)callInviteReceived:(TVOCallInvite *)callInvite {
 
     // If the application is not in the foreground, post a local notification
     if ([[UIApplication sharedApplication] applicationState] != UIApplicationStateActive) {
-        UIApplication* app = [UIApplication sharedApplication];
-        UILocalNotification* notification = [[UILocalNotification alloc] init];
-        notification.alertBody = [NSString stringWithFormat:@"Incoming Call from %@", callInvite.from];
-
-        [app presentLocalNotificationNow:notification];
+        UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
+        UNMutableNotificationContent *content = [UNMutableNotificationContent new];
+        content.title = @"Incoming Call";
+        content.body = [NSString stringWithFormat:@"Call Invite from %@", callInvite.from];
+        content.sound = [UNNotificationSound defaultSound];
+        
+        UNTimeIntervalNotificationTrigger *trigger = [UNTimeIntervalNotificationTrigger triggerWithTimeInterval:1 repeats:NO];
```

Comment:
Is it delaying the notification by 1 sec?

Reply:
For some reason the `UserNotification` library doesn't allow `0` or provide a "fire immediately" method. People seeing the same issue: https://forums.developer.apple.com/thread/49889.
It seems that you can pass `nil` for the trigger. I'll try this.